alacritty/src/display/hint.rs

```diff
@@ -70,21 +73,40 @@ impl HintState {
             None => return,
         };
 
-        // Find visible matches.
-        self.matches.0 = hint.regex.with_compiled(|regex| {
-            let mut matches = RegexMatches::new(term, regex);
-
-            // Apply post-processing and search for sub-matches if necessary.
-            if hint.post_processing {
-                matches
-                    .drain(..)
-                    .map(|rm| HintPostProcessor::new(term, regex, rm).collect::<Vec<_>>())
-                    .flatten()
-                    .collect()
-            } else {
-                matches.0
+        let mut hint_matches: Vec<Match> = Vec::new();
+
+        // Find hyperlinks.
+        if hint.hyperlinks {
+            let mut visited = HashSet::new();
```

Comment:
Not sure what you mean by "visited" here? There is no tracking of hyperlinks that have been clicked, the concept of "visited" seems misleading.